docs/reference/components/sinks/console.cue

```diff
@@ -0,0 +1,72 @@
+package metadata
+
+components: sinks: console: {
+	title:             "Console"
+	short_description: "Streams log and metric events to [standard output streams][urls.standard_streams], such as [STDOUT][urls.stdout] and [STDERR][urls.stderr]."
+	long_description:  "Streams log and metric events to [standard output streams][urls.standard_streams], such as [STDOUT][urls.stdout] and [STDERR][urls.stderr]."
+
+	classes: {
+		commonly_used: false
+		function:      "test"
+		service_providers: []
+	}
+
+	features: {
+		batch: {
+			enabled:      false
+			common:       false
+			max_bytes:    null
+			max_events:   null
+			timeout_secs: 0
+		}
```

Comment:
This is my fault, but this should probably be:

```
batch: enabled: false
```

And that's it.

Reply:
I think I had check-docs failing with missing values for common, max_bytes, etc.

Reply:
`check-docs` fails with:
```
components.sinks.console.features.batch.common: incomplete value bool
components.sinks.console.features.batch.max_bytes: incomplete value >=0 & int | null
components.sinks.console.features.batch.max_events: incomplete value >=0 & int | null
components.sinks.console.features.batch.timeout_secs: incomplete value >=0 & <=255 & int
```